Add second blog post and update first blog title

anushkasignoz opened the pull request:
- Added new blog: Cloud vs Self-Hosted: Which Deployment Model Is Right For You?
- Updated title of query performance improvement blog for lobste.rs

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+
+## Introduction
+
+The Logs Explorer page in SigNoz enables developers to filter, examine, and analyze logs. There are two ways to filter the logs and each way has 3 different views available in Logs Explorer which include:
```
Search is just to add basic filters and to apply expanded operations with filters we have Query Builder

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+
+---
+
+**Settings**
```
We should be able to hyperlink setting section to this page

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+2. **Default**: It provides a balanced view by formatting a raw log based on different attributes to make it more readable.
+3. **Column**: It organizes the logs data into different attribute Columns to provide a tabular view of logs.
+
+<figure data-zoomable align='center'>
```
GIF here shows adding and removing columns.
The example GIF should show all three views.

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+
+### Time Series View
+
+The Time Series View provides a graphical representation of logs data over time based on the filters applied.
```
based on filters or operations applied

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+
+### Table View
+
+The Table View provides a tabular representation of the count logs data based on the filters applied.
```
Here it's been said that table view shows count logs but it can be avg for duration of operation or any other operation.

makeavish commented on docs/product-features/logs-explorer.md:
```diff
@@ -0,0 +1,324 @@
+When you click on a log line in any format it will open up a Log Detail panel which gives a comprehensive view of your log. This is only available in the List View.
+
+<figure data-zoomable align='center'>
+    <img src="/img/docs/product-features/logs-explorer/logs-explorer-log-details.gif" alt="Log Details Panel in Logs Explorer"/>
```
Can we use an image rather than a GIF? It's easy for the user to understand that panel opens on click on a log. On GIF user is unable to get good view of panel.

makeavish commented:
@Calm-Rock Please resolve conflicts and merge

anushkasignoz committed:
Add second blog post and update first blog title

- Added new blog: Cloud vs Self-Hosted: Which Deployment Model Is Right For You?
- Updated title of query performance improvement blog
